# Odometry node config
leo_localization:
   robot_dimensions:
      wheelbase: 0.360
      wheel_diameter: 0.13

   odometry:
      frame_id: "odom"
      use_initial_pose: false
      gaussian: false
      alpha1RotRot: 0.01
      alpha2RotTrans: 0.01   
      alpha3TransTrans: 0.01
      alpha4TransRot: 0.01
      topics:
         publish_tf: false
         odomPublisher: "/odometry/odom"
         distPublisher: "/odometry/pose_distribution"
         plcSubscriber: "/velocity_odom"
         resetSubscriber: "/odometry/reset"
         initialPoseSubscriber: "/odometry/initial_pose"
